src/swapCostCalculator.ts

```diff
@@ -25,7 +25,7 @@ export class SwapCostCalculator {
         tokenAddress: string,
         tokenDecimals: number,
         gasPriceWei: BigNumber,
-        swapGas: BigNumber = BigNumber.from('35000')
```

Comment:
I think we agreed that in the end changing this value doesn't have any benefits so can we keep it that same. Thanks.